_pages/2019/x/psets/3/index.md

```diff
@@ -8,8 +8,8 @@ layout: 2019/x
 
 1. Submit, using CS50 Lab, [Whodunit](https://lab.cs50.io/cs50/labs/2019/x/whodunit/)
 1. Submit, using CS50 IDE, one of:
-  * [this version of Resize](https://docs.cs50.net/2019/x/psets/3/resize/less/resize.html), if feeling less comfortable
-  * [this version of Resize](https://docs.cs50.net/2019/x/psets/3/resize/more/resize.html), if feeling more comfortable
+   * [this version of Resize](https://docs.cs50.net/2019/x/psets/3/resize/less/resize.html), if feeling less comfortable
+   * [this version of Resize](https://docs.cs50.net/2019/x/psets/3/resize/more/resize.html), if feeling more comfortable
 1. Submit [Recover](https://docs.cs50.net/2019/x/psets/3/recover/recover.html), using CS50 IDE
```

Comment:
Although this was rendering correctly on your production site, it wasn't rendering correctly on GitHub.

![image](https://user-images.githubusercontent.com/31348972/72151162-34201580-33b0-11ea-9f3a-d4c4aa8e81c7.png)

And that's one advantage of following the markdown rules. It ensures that all md parsers will behave the same.

Violating rules, as this example shows, can lead to different behaviors on different parsers.